hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java

```diff
@@ -122,18 +122,30 @@
     FRONTEND_SSL_PORT("frontend.ssl.port", 8443),
     FRONTEND_SSL_CLIENT_AUTH_MODE("frontend.ssl.client.auth.mode", "not_requested"),
     FRONTEND_SSL_PROTOCOL("frontend.ssl.protocol", "TLS"),
+
+    FRONTEND_SSL_KEYSTORE_DEFAULT_JVM("frontend.ssl.keystore.default.jvm", true),
```

Comment:
I am not sure whether this name is comprehensible. 
Maybe `frontend.ssl.keystore.jvm.based` would be more appropriate...?

Reply:
I wonder If this should be a string type property (enum-like), say:
`frontend.ssl.keystore.source`, with default value equal to `jvmDefault` (the other option would be `provided`).
In this case there would be no possibility for empty value (both are `false`), plus no need for guessing which option has higher priority when both are set to `true`.

Reply:
Done

Reply:
@dankraw since we load trust store from JRE path maybe `{consumer,frontend}.ssl.{key,trust}store.source.jre` would be better name for this property?

Reply:
`jre` sounds alright. :)